skyfield/contrib/almanac2/almanac2_examples.py

```diff
@@ -0,0 +1,48 @@
+from skyfield.api import Loader, Topos, EarthSatellite, Star
+from almanac2 import seasons, moon_phases, meridian_transits, culminations, twilights, risings_settings
+
+load = Loader(r'C:\Users\Josh\Scripts\Skyfield_Data')
```

Comment:
maybe a more explicit line 4 would be welcome ?

Reply:
Thanks for pointing this out, I should have mentioned it before. The tests and the example script require de430t.bsp, which is quite large (130 MB). I didn't want that to automatically start downloading to anyone's skyfield/contrib/almanac2 directory, so in order to run ``test_almanac2.py`` or ``almanac2_examples.py`` you'll have to edit them so that the Loader instance uses whichever directory you keep your .bsp files. I don't know of a way to avoid automatically downloading de430t.bsp and still allow those scripts to run without needing modification first. I just changed this so that there's an empty string instead of the path from my computer.

You should still be able to import from ``almanac2.py`` without any modifications.